README.md

```diff
@@ -1,2 +1,132 @@
 # calendar-date
-Immutable object to represent a calendar date with zero dependencies.
+
+A calendar date is a date without time information, e.g. "2020-01-01".
+This library provides an immutable object to represent and work with a calendar date.
+
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)
+
+## Why another date library?
+
+There are a lot of date libraries in the javascript world, but there is currently no good support for representing a calendar date.
+Most SQL Databases like MySQL and PostgreSQL have a datatype for representing a date without time information.
+
+Using the build-in date object for a calendar date poses a number of disadvantages:
+- Overhead by including time and timezone information in the representation.
+- Error prone when working with different timezones and converting from and to Strings.
+  For example `new Date(2020, 0, 1)` and `new Date('2020-01-01')` return date objects with a difference of one hour.
+  If you would call `toISOString()` on the first date object it would return 2019-12-31T23:00:00.000Z.
```

Comment:
```suggestion
  If you would call `toISOString()` on the first date object it would return 2019-12-31T23:00:00.000Z. For the second you get 2020-01-01T01:00:00.000Z.
```